ledger/light/proc/get_request_test.go

```diff
@@ -84,6 +84,52 @@ func TestGetRequest_Proceed(t *testing.T) {
 		assert.NoError(t, err)
 	})
 
+	t.Run("Passing request on light", func(t *testing.T) {
+		setup()
+		defer mc.Finish()
+
+		records.ForIDMock.Return(record.Material{}, object.ErrNotFound)
+
+		expectedTarget := insolar.NewReference(gen.ID())
+
+		coordinator.IsBeyondLimitMock.Return(false, nil)
+		expectedJetID := gen.ID()
+		fetcher.FetchMock.Return(&expectedJetID, nil)
+		coordinator.LightExecutorForJetMock.Inspect(func(ctx context.Context, jetID insolar.ID, pulse insolar.PulseNumber) {
+			assert.Equal(t, jetID, expectedJetID)
+		}).Return(expectedTarget, nil)
+		requestID := gen.IDWithPulse(gen.PulseNumber())
+		expectedUpdateJet, _ := payload.NewMessage(&payload.UpdateJet{
+			Pulse: requestID.Pulse(),
+			JetID: insolar.JetID(expectedJetID),
+		})
+
+		meta := payload.Meta{
+			Sender: gen.Reference(),
+		}
+		buf, _ := meta.Marshal()
+		expectedPass, _ := payload.NewMessage(&payload.Pass{
+			Origin: buf,
+		})
+
+		sender.SendTargetMock.Inspect(func(ctx context.Context, msg *message.Message, target insolar.Reference) {
+			if sender.SendTargetAfterCounter() == 0 {
```

Comment:
Let's make a type switch instead of locking of operation order.

Reply:
fixed